tutorial/712_DataSmoothing/isolines.h

```diff
@@ -0,0 +1,90 @@
+
+#include <vector>
+#include <limits>
+#include <stdlib.h>
+
+#include <igl/remove_unreferenced.h>
+
+
+static void isolines(const Eigen::MatrixXd& V,
```

Comment:
This could be useful in general. It'd be cool to (eventually) pull this out and template this into a .h/.cpp file.